### ISSUE TITLE
Data structure & editor view to edit the words

### ISSUE THREAD

**Issue by samuelhackwill** [samuelhackwill/le-tiret-du-six#55]
- [x] We need an editor-like component to edit the db of words.
- [x] We need a parser with specific rules to populate the DB from a "plainsam" file. Probably as such :  
   
``` javascript
   Line // first line is word name
   blablablalbabalabla // second line is the text body (description of the word)
   "pierre qui roule n'amasse pas mousse" // third line is the drolatique citation
   (André Breton) // fourth line is the author of the citation.

```
- [x] We need a validated DB to store the words, probably an array of objects with a name, text, citation, author, and status of individual letters

**Comment by samuelhackwill** [comment]
first task implemented in e733309

**Comment by samuelhackwill** [comment]
minor bug : you can't come out of editing mode if you try to edit both dbs at the same time. Not critical though so won't fix

**Comment by samuelhackwill** [comment]
middle task implemented in ec1c8b7
